x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/use_output_form.tsx

```diff
@@ -327,18 +334,43 @@ export function useOutputForm(onSucess: () => void, output?: Output, defaultOupu
 
   const proxyIdInput = useInput(output?.proxy_id ?? '', () => undefined, isDisabled('proxy_id'));
 
+  const extractDefaultDynamicKafkaTopics = (
```

Comment:
it would be nice to cover the logic in unit tests

Reply:
I'll add some tests for this. Thanks Julia!